### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by sindresorhus**
I used this generator to make `alfred-dark-mode` and noticed some minor issues.

- Upgrade the generator system
- Fix name generation
- Simplify URL parsing
- Remove `alfred-` from bundleID

### CODE REVIEW

File: app/utils.js [hunk 1]
```diff
@@ -2,37 +2,30 @@
...
 	return id;
 };
 
 exports.bundleId = props => {
-	let website = props.website;
-
-	if (!/^https?:\/\//.test(website)) {
-		// Make sure website starts with the protocol. `Url.parse` could not handle `test.com` for instance.
-		website = `http://${website}`;
-	}
```
Comment (sindresorhus) [inline app/utils.js:27]: This was moot as the prompt `filter` already normalizes the URL and adds a protocol.

File: app/index.js [hunk 2]
```diff
@@ -2,20 +2,27 @@
...
-				filter: x => moduleName.slugify(x)
+				filter: x => {
+					let name = moduleName.slugify(x);
+
+					if (!/^alfred-/.test(name)) {
+						name = `alfred-${name}`;
+					}
+
+					return name;
+				}
```
Comment (SamVerschueren) [inline app/index.js:25]: Good idea to always prefix the module name!

**Comment by SamVerschueren** [comment]
Woot! Thanks for the improvements, much appreciated!

File: app/index.js (hunk 2 above)
Comment (sindresorhus) [reply app/index.js:25]: Yes, I forgot and ended up with the wrong name in many places.

**Pull request merged**